tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -449,6 +450,143 @@ function ( Request $request ) use ( $property_id, $webdatastream_id, $measuremen
 		);
 	}
 
+	public function test_handle_provisioning_callback__with_enhancedMeasurement_streamEnabled() {
+		$this->enable_feature( 'enhancedMeasurement' );
+		$account_id       = '12345678';
+		$property_id      = '1001';
+		$webdatastream_id = '2001';
+		$measurement_id   = '1A2BCD345E';
+
+		$options = new Options( $this->context );
+		$options->set(
+			Settings::OPTION,
+			array(
+				'accountID'       => $account_id,
+				'propertyID'      => '',
+				'webDataStreamID' => '',
+				'measurementID'   => '',
+			)
+		);
+
+		// TODO: Rework this giant handler into one composed of per-request handlers.
```

Comment:
This is a bit verbose because we don't have an existing helper for composing Google HTTP handlers in a more elegant way. I realize there is already a dedicated method in this class for an enhanced measurement handler but this only covers the one request which will not work for this test which needs to be able to fake creating a property, and datastream.